### PR TITLE
Add ExtraFIPSEnv launch configuration

### PR DESCRIPTION
**Pull request by neoaggelos**
### Summary

Add `extraFIPSEnv` launch configuration, to control the `fips-env` file

### CODE REVIEW

File: pkg/k8sinit/apply.go [hunk 1]
```diff
@@ -67,34 +67,37 @@ func (s *launcherScope) applyPart(ctx context.Context, c *Configuration) error {
...
-		{configFile: "etcd-env", restartService: "etcd", args: c.ExtraEtcdEnv},
-		{configFile: "flanneld", restartService: "flanneld", args: c.ExtraFlanneldArgs},
-		{configFile: "flanneld-env", restartService: "flanneld", args: c.ExtraFlanneldEnv},
-		{configFile: "cni-env", restartService: "", args: c.ExtraCNIEnv},
+		{configFile: "kube-apiserver", restartServices: []string{"kubelite"}, args: c.ExtraKubeAPIServerArgs},
+		{configFile: "kubelet", restartServices: []string{"kubelite"}, args: c.ExtraKubeletArgs},
+		{configFile: "kube-proxy", restartServices: []string{"kubelite"}, args: c.ExtraKubeProxyArgs},
+		{configFile: "kube-controller-manager", restartServices: []string{"kubelite"}, args: c.ExtraKubeControllerManagerArgs},
+		{configFile: "kube-scheduler", restartServices: []string{"kubelite"}, args: c.ExtraKubeSchedulerArgs},
+		{configFile: "kubelite-env", restartServices: []string{"kubelite"}, args: c.ExtraKubeliteEnv},
```
Comment (ktsakalozos) [inline pkg/k8sinit/apply.go:79]: I do not think we have all these env files.

Comment (neoaggelos) [reply pkg/k8sinit/apply.go:79]: these are argument files, not env vars (except for `kubelite-env`)

Comment (ktsakalozos) [reply pkg/k8sinit/apply.go:79]: I could not find the etcd-env file, but I might be wrong.

Comment (neoaggelos) [reply pkg/k8sinit/apply.go:79]: True, we don't really read this at the moment, this is resolved in https://github.com/canonical/microk8s/pull/4135)

**Pull request merged**